source/_views/default.html

```diff
@@ -12,69 +12,71 @@
-                                    target="_blank"
-                                >
-                                    &nbsp;
-                                </a>
-                            {% endif %}
-                            {% if site.github %}
-                                <a
-                                    class="github"
-                                    href="https://github.com/{{ site.github }}"
-                                    rel="me"
-                                    rel="me" title="{{ site.twitter }} github profile"
-                                    target="_blank"
-                                >
-                                    &nbsp;
-                                </a>
-                            {% endif %}
-                        <ul>
-                        </ul>
+                        {% if site.twitter %}
+                        <a href="https://twitter.com/{{ site.twitter }}" rel="me" title="{{ site.twitter }} twitter profile" target="_blank"><i class="fa fa-fw fa-twitter"></i><span>Twitter</span></a>
+                        {% endif %}
+
+                        {% if site.github %}
+                        <a href="https://github.com/{{ site.github }}" rel="me" title="{{ site.twitter }} github profile" target="_blank"><i class="fa fa-fw fa-github"></i><span>Github</span></a>
+                        {% endif %}
                     </div>
+                </nav>
+                <button class="close-button" id="close-button">Close Menu</button>
+                <div class="morph-shape" id="morph-shape" data-morph-open="M-7.312,0H15c0,0,66,113.339,66,399.5C81,664.006,15,800,15,800H-7.312V0z;M-7.312,0H100c0,0,0,113.839,0,400c0,264.506,0,400,0,400H-7.312V0z">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 100 800" preserveAspectRatio="none">
+                        <path d="M-7.312,0H0c0,0,0,113.839,0,400c0,264.506,0,400,0,400h-7.312V0z"/>
+                    </svg>
                 </div>
             </div>
-        </header>
-        <div class="mainContent container">
-            <div class="row-fluid">
-                {#
-                <div class="span11 align-center">
-                    <h1>{{ site.title }}</h1>
-                    <p class="description"> {{ site.description }} </p>
-                </div>
-                #}
-                <div class="span11">
-                    {% block content_wrapper %}{% block content %}{% endblock %}{% endblock %}
+            <button class="menu-button" id="open-button">Open Menu</button>
+            
+            <div class="content-wrap">
+                <div class="content">
+                    <div class="top left">
+                        <p id="shield-container" class="logo-cont left" style="width: 200px; padding: 0; margin: 0; height: 200px;
+"></p>
+                        <h1 class="left text-center">Marco Pivetta <span>(aka Ocramius)</span></h1>
```

Comment:
This cannot be the H1, as that's not the heading for every article (it may be for the homepage only)